src/database/migrations/20220920054800-create-launches.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("launches", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      user_id: {
+        references: {
+          model: "users",
+          key: "id",
+        },
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      site_id: {
+        references: {
+          model: "users",
```

Comment:
I think this should be `sites`, not `users`?

Reply:
25e5c88 oh dear, thanks for catching this!